torch_geometric/nn/conv/gat_conv.py

```diff
@@ -195,6 +215,10 @@ def reset_parameters(self):
             self.lin_dst.reset_parameters()
         if self.lin_edge is not None:
             self.lin_edge.reset_parameters()
+        if isinstance(self.lin_res, Linear):
+            self.lin_res.reset_parameters()
+            if self.lin_res.bias is not None:
```

Comment:
I think `zeros` also works if self.lin_res.bias is None

Reply:
You are right, we can save it here.